src/client/Microsoft.Identity.Client/Instance/Discovery/InstanceDiscoveryManager.cs

```diff
@@ -114,7 +114,8 @@ internal class InstanceDiscoveryManager : IInstanceDiscoveryManager
 
         public async Task<InstanceDiscoveryMetadataEntry> GetMetadataEntryAsync(
             string authority, 
-            RequestContext requestContext)
+            RequestContext requestContext,
+            bool withAzureRegion = false)
```

Comment:
isn't this flag available on the RequestContext? Similar to requestContext.ServiceBundle.Config.AuthorityInfo ?

Reply:
Moved it into request context.